webview/src/experiments/util/buildColumns.tsx

```diff
@@ -51,7 +51,7 @@ const getMainColumnProperties = (
         CellContext<Column, CellValue>
       >,
       header: () => <TimestampHeader />,
-      id: ColumnType.TIMESTAMP
+      id: 'Created'
```

Comment:
[F] This mismatched with the value provided in `columnOrder` which initially placed the column at the end of the table until any reordering added `id: 'timestamp'` to the table (and extension's) state.

Reply:
tricky one to track down